eng/helix/helix.proj

```diff
@@ -75,10 +83,6 @@
     <HelixProperties Condition="'$(RunQuarantinedTests)' != 'true'" Include="runType" Value="unquarantined" />
   </ItemGroup>
 
-  <ItemGroup Condition="Exists('$(HelixTestConfigurationFilePath)')">
-    <HelixCorrelationPayload Include="$(HelixTestConfigurationFilePath)" AsArchive="false" />
```

Comment:
I'm assuming the helix sdk is doing this for us now?

Reply:
> I'm assuming the helix sdk is doing this for us now?

Yes, this line just created a duplicate item.